contrib/jemalloc/src/tcache.c

```diff
@@ -493,10 +493,10 @@ tcache_create_explicit(tsd_t *tsd) {
 	stack_offset = size;
 	size += stack_nelms * sizeof(void *);
 	/* Avoid false cacheline sharing. */
-	size = sz_sa2u(size, CACHELINE);
+	size = ROUND_SIZE(sz_sa2u(size, CACHELINE));
 
-	tcache = ipallocztm(tsd_tsdn(tsd), size, CACHELINE, true, NULL, true,
-	    arena_get(TSDN_NULL, 0, true));
+	tcache = BOUND_PTR(ipallocztm(tsd_tsdn(tsd), size, CACHELINE, true,
```

Comment:
There is a check for tcache == NULL below, so I guess we should move the bounding below the check since it will crash otherwise.

Reply:
I thought so but BOUND_PTR handles that case with a ternary.

Reply:
(since it's something you need in _every_ instance inside jemalloc)

Reply:
I should have checked BOUND_PTR first...